src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -1,49 +1,42 @@
-import {AlterationTypeConstants, AnnotatedMutation, AnnotatedNumericGeneMolecularData} from "../ResultsViewPageStore";
+import { MobxPromise } from "mobxpromise";
+import { fromPromise, FULFILLED } from "mobx-utils";
 import numeral from "numeral";
-import {getJitterForCase} from "../../../shared/components/plots/PlotUtils";
-import {isSampleProfiled} from "../../../shared/lib/isSampleProfiled";
+import * as React from "react";
+import { ClinicalAttribute, ClinicalData, Gene, MolecularProfile, Mutation, NumericGeneMolecularData, Sample } from "../../../shared/api/generated/CBioPortalAPI";
+import { remoteData } from "../../../shared/api/remoteData";
+import { getSampleViewUrl } from "../../../shared/api/urls";
 import GenesetMolecularDataCache from "../../../shared/cache/GenesetMolecularDataCache";
-import {GenesetMolecularData} from "../../../shared/api/generated/CBioPortalAPIInternal";
+import { GenesetMolecularData, TreatmentMolecularData } from "../../../shared/api/generated/CBioPortalAPIInternal";
 import {MUTATION_COUNT} from "../../studyView/StudyViewPageStore";
 import ClinicalDataCache from "../../../shared/cache/ClinicalDataCache";
+import TreatmentMolecularDataCache, { TreatmentMolecularDataEnhanced } from "../../../shared/cache/TreatmentMolecularDataCache";
+import { getOncoprintMutationType, OncoprintMutationType, selectDisplayValue } from "../../../shared/components/oncoprint/DataUtils";
+import { CNA_COLOR_AMP, CNA_COLOR_HOMDEL, DEFAULT_GREY, MUT_COLOR_FUSION, MUT_COLOR_INFRAME, MUT_COLOR_INFRAME_PASSENGER, MUT_COLOR_MISSENSE, MUT_COLOR_MISSENSE_PASSENGER, MUT_COLOR_OTHER, MUT_COLOR_PROMOTER, MUT_COLOR_TRUNC, MUT_COLOR_TRUNC_PASSENGER } from "../../../shared/components/oncoprint/geneticrules";
+import { IBoxScatterPlotData } from "../../../shared/components/plots/BoxScatterPlot";
+import { getJitterForCase, dataPointIsLimited } from "../../../shared/components/plots/PlotUtils";
+import { IBaseScatterPlotData } from "../../../shared/components/plots/ScatterPlot";
+import { isSampleProfiled } from "../../../shared/lib/isSampleProfiled";
+import MobxPromiseCache from "../../../shared/lib/MobxPromiseCache";
+import { stringListToIndexSet } from "../../../shared/lib/StringUtils";
+import { AlterationTypeConstants, AnnotatedMutation, AnnotatedNumericGeneMolecularData } from "../ResultsViewPageStore";
+import { CoverageInformation } from "../ResultsViewPageStoreUtils";
+import { AxisMenuSelection, MutationCountBy, NONE_SELECTED_OPTION_NUMERICAL_VALUE, NONE_SELECTED_OPTION_STRING_VALUE, ViewType, PlotType, UtilitiesMenuSelection } from "./PlotsTab";
+import { IBaseWaterfallPlotData } from "shared/components/plots/WaterfallPlot";
 
 export const CLIN_ATTR_DATA_TYPE = "clinical_attribute";
+export const BOUNDARY_VALUE_NAME = "Limit"; // "... value": name to indicate values that are > or < than a certain value (e.g. "">8.00")
 export const GENESET_DATA_TYPE = "GENESET_SCORE";
+export const TREATMENT_DATA_TYPE = "TREATMENT_RESPONSE";
```

Comment:
AlterationTypeConstants

Reply:
oh looks like you used it below, so just delete this

Reply:
Removed.